guides/user/kubernetes-v2/deploy-manifest/index.md

```diff
@@ -0,0 +1,173 @@
+
+## Specifying Your Manifest
+
+Depending on your needs, there is more than one way to specify the manifest
+that you want to deploy:
+
+* [Statically: directly in the pipeline](#statically-specified-manifests)
+* [Dynamically: bound at runtime using an
+  artifact](#dynamically-specified-manifests)
+
+In either case, we want to start by selecting the __Deploy (Manifest)__ stage
+from the stage selector as shown here:
+
+{%
+  include
+  figure
+  image_path="./deploy-manifest.png"
+%}
+
+> :warning: The regular __Deploy__ stage is used to deploy more opinionated
+> "Server Groups" in Spinnaker, using either another provider, or the
+> Kubernetes V1 provider.
+
+### Statically specified manifests
+
+If you know ahead of time what you expect to deploy a certain manifest (even if
+you don't know what version of your docker image it will run) you can declare
+it directly in the pipeline the manifest specification.
+
+{%
+  include
+  figure
+  image_path="./in-pipeline.png"
+  caption="Notice that by selecting \"Text\" as the __Manifest Source__, we get
+  to enter the Manifest YAML by hand."
+%}
+
+Of course, if you are generating your pipeline definitions rather than entering
+them into the UI, the stage definition would look more like:
+
+```json
+{
+  "name": "Deploy my manifest",   // human-readable name
+  "type": "deployManifest",       // tells orchestration engine what to run
+  "account": "nudge",             // account (k8s cluster) to deploy to
+  "cloudProvider": "kubernetes",
+  "source": "text",
+  "manifest": {
+                                  // manifest contents go here
+  },
+  "moniker": {                    // used to specify app & cluster for grouping
+                                  // resources in UI
+    "app": "xnat",
+    "cluster": "c7",
+  }
+}
+```
+
+### Dynamically specified manifests
+
+If you are storing your manifests outside of Spinnaker's pipeline repository,
+or want a single deploy stage to be able to deploy a variety of manifests, you
+can specify your manifest using an [Artifact](/reference/artifacts).
+
+The idea is: artifacts in Spinnaker allow you to reference remote, deployable
+resources. In our case, artifacts will have to reference a text file
+(containing a Manifest specification) stored in GitHub, or an object store
+(like GCS).
+
+There are already some guides describing how to trigger off of changes to these
+artifacts, and supply them to pipelines. See:
+
+* [Consuming GitHub Artifacts](/guides/user/artifacts/github)
+* [Consuming GCS Artifacts](/guides/user/artifacts/gcs)
+
+Assuming you have declared an expected artifact upstream to your Deploy
+manifest stage, you can reference it in the Deploy configuration:
+
+{%
+  include
+  figure
+  image_path="./in-artifact.png"
+  caption="Notice that by selecting \"Artifact\" as the __Manifest Source__, we
+  get to pick which upstream artifact to deploy."
+%}
+
+> __☞ Note__: Based on how you configured your artifact accounts, make sure that
```

Comment:
Is "Based on how...accounts," necessary?

Reply:
No, removed